src/myspell/myspell_checker.cpp

```diff
@@ -44,10 +44,12 @@
 
 /* built against hunspell 1.2.2 on 2008-04-12 */
 
-#ifdef HUNSPELL_STATIC
-#include "hunspell.hxx"
-#else
 #include <hunspell/hunspell.hxx>
+
+/* Some version of hunspell (1.4.x) don't have this defined. */
```

Comment:
In that context is is "pluriel" because gender is undetermined. I know, French grammar is complex.

Pushed a fixed commit.